pkgs/development/tools/fprettify/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, python3Packages, fetchFromGitHub }:
+
+python3Packages.buildPythonApplication rec {
+    pname = "fprettify";
+    version = "0.3.7";
+
+    src = fetchFromGitHub {
+        owner = "pseewald";
+        repo = pname;
+        rev = "v${version}";
+        sha256 = "17v52rylmsy3m3j5fcb972flazykz2rvczqfh8mxvikvd6454zyj";
+    };
+
+    propagatedBuildInputs = with python3Packages; [
+        configargparse
+    ];
+
+    # The checks use subprocess.py, which fails with a FileNotFoundError
+    # on Linux. On macOS, it works just fine.
+    doCheck = false;
```

Comment:
```suggestion
    doCheck = stdenvNoCC.isDarwin;
```

Reply:
This is not necessary anymore, thanks to your fix. :)